roman_numeral_converter.rb

```diff
@@ -0,0 +1,41 @@
+class RomanArabicConverter
+
+  ROMAN_TO_ARABIC_MAP = {
+    "M"=>1000,
+    "CM"=>900,
+    "D"=>500,
+    "CD"=>400,
+    "C"=>100,
+    "XC"=>90,
+    "L"=>50,
+    "XL"=>40,
+    "X"=>10,
+    "IX"=>9,
+    "V"=>5,
+    "IV"=>4,
+    "I"=>1
+  }
+
```

Comment:
Without an initialize method, this might as well be a module. Even better, allow the user to DI a different set of `ROMAN_TO_ARABIC_MAP` at will via the initialize method.